eng/common/pipelines/ci.yml

```diff
@@ -41,20 +41,24 @@ extends:
           Condition: eq('true', stageDependencies.InitStage.outputs['InitJob.InitStep.RunCore'])
 
       # Run eng test stage if RunEng == true
-      - stage: TestEng
-        displayName: Test Eng
+      - stage: EngTest
+        displayName: Eng - Test
         dependsOn: InitStage
         condition: eq('true', stageDependencies.InitStage.outputs['InitJob.InitStep.RunEng'])
         jobs:
-          - job: TestEng
+          - job: EngTest
+            displayName: Eng - Test
             pool:
               name: $(LINUXPOOL)
               image: $(LINUXVMIMAGE)
               os: linux
             steps:
               - pwsh: |
-                  Install-Module -Name Pester -Force -SkipPublisherCheck
+                  Write-Host "Installing Pester"
```

Comment:
I'll just comment here too, this is ok if we are just keeping those powershell script to only be for the pipelines but I would be very careful about using any powershell for anything that would involve local building. It is yet another tool that needs to be installed by contributors.

I personally would prefer if this repo would remove all deps on powershell but if that's better for eng sys i'm ok keeping it for pipelines(and csharp packages though same problem as above)

Reply:
We have a lot of investment in ps tooling.  I am happy to find some investment in a longer term swap but in the short term I don't think we have time to undertake this.

Reply:
Yeah I'm fine with keeping existing behavior for now(and figuring out a long term migration) with PS (including this script as it's already written) but in the future it would be nice to consider not having PS for new scripts